src/Controllers/DatabaseController.js

```diff
@@ -112,13 +112,14 @@ DatabaseController.prototype.untransformObject = function(
     return object;
   }
 
-  delete object.authData;
```

Comment:
I thought the whole reason for the change in the first place was to strip out legacy session tokens where they still exist in the DB?